Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cargo-workspace-version"
+version = "0.0.3"
```

Comment:
I think we can version this differently since it's now a standalone project. Let's revert back to `0.0.1`?